head.html

```diff
@@ -134,6 +134,36 @@
     }
   }
 
+  function fetchProducts() {
+    const div = getSection();
+    const insertAfter = getSection(2);
+    if (insertAfter) {
+      const r = /^Products\: (.*)$/gm.exec(div.innerText);
```

Comment:
```suggestion
      const r = /^Products\: (.*)$/gmi.exec(div.innerText);
```
Make regexp case-insensitive.